control/control.go

```diff
@@ -271,6 +271,7 @@ func (p *pluginControl) Load(rp *core.RequestedPlugin) (core.CatalogedPlugin, se
 	if serr != nil {
 		return nil, serr
 	}
+	details.IsAutoLoaded = rp.AutoLoaded()
```

Comment:
Is this assignment necessary? I think details.IsAutoLoaded is already set correctly in returnPluginDetails() call on L347.